zilliqa/Cargo.toml

```diff
@@ -61,7 +61,7 @@ revm-inspectors = { version = "0.8.1", features = ["js-tracer"] }
 rusqlite = { version = "0.32.1", features = ["bundled", "trace"] }
 serde = { version = "1.0.210", features = ["derive", "rc"] }
 serde_bytes = "0.11.14"
-serde_json = { version = "1.0.128", features = ["raw_value"] }
+serde_json = { version = "1.0.132", features = ["raw_value","arbitrary_precision"] }
```

Comment:
Is this feature actually required to get `serde_json` to serialize `u128`s? I had one experience in the past (3+ years ago) where this feature interacted badly with some other library that used serde in a weird and hard-to-spot way that makes me wary :)

Reply:
Yep, without it it appears to convert to a float (Number(2.1e28)).
It's a bit tricky to tell what's going on because without `arbitrary_precision` we also can't deserialise values, since `as_u128() ` just returns None, so it's not really possible to test it.

Reply:
Right, it looks like the specific problem is that without `arbitrary_precision`, the values above u64::MAX get serialised to a float, and `as_u128()` quite reasonably don't deserialise a float

Reply:
Huh fair enough, thanks for double-checking :)